internal/appmetrics/counters.go

```diff
@@ -31,4 +31,30 @@ var (
 		Name: "devices_api_blackbook_request_success_ops_total",
 		Help: "Total successful Blackbook used",
 	})
+	// Chat GPT Metrics
+	OpenAITotalCallsOps = promauto.NewCounter(prometheus.CounterOpts{
+		Name:      "devices_api_openai_chatgpt_ops_total",
+		Help:      "Total number of calls to Open AI ChatGPT",
+		Namespace: "devices_api",
+		Subsystem: "error_codes",
+	})
+	OpenAITotalFailedCallsOps = promauto.NewCounter(prometheus.CounterOpts{
+		Name:      "devices_api_openai_chatgpt_ops_failed_calls_total",
+		Help:      "Total number of failed calls to Open AI ChatGPT",
+		Namespace: "devices_api",
+		Subsystem: "error_codes",
+	})
+	OpenAITotalTokensUsedOps = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
Oh, do we ever use this? I remember there are token counts from the input and output, and I figured we would add those numbers up and increase the counter by that much.

Reply:
There is total token count being returned as well. I have made changes to add the values to `OpenAITotalTokensUsedOps`  now

Reply:
Much better now that it is being used.